src/Aquifer.Common/Utilities/BibleTextUtilities.cs

```diff
@@ -0,0 +1,42 @@
+using Aquifer.Common.Services.Caching;
+using Aquifer.Data;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.Common.Utilities;
+
+public static class BibleTextUtilities
+{
+    public static async Task<IEnumerable<Chapter>> GetTextsByBibleId(int bookNumber, int chapter, int verse,
```

Comment:
I think this would ideally use the VerseId from the Verses table, or there'd be an overload of this that took it.